tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -466,6 +483,10 @@ module.exports = {
       selector: '//button/span[.="Save"]',
       locateStrategy: 'xpath'
     },
+    publicLinkDeleteExpirationDateButton: {
+      selector: '//*[@uk-tooltip="Remove expiration date"]',
```

Comment:
in general we try to rely on ids or classes as texts could change.
when either is missing, we add them to the original code

Reply:
Thanks for the input, this is actually very useful information! :-) I changed all the selectors in the publicLinksDialog spec to not use texts anymore and added appropriate ids/classes to the DOM elements.